components/nexus-repository/src/main/java/org/sonatype/nexus/repository/storage/StorageFacetImpl.java

```diff
@@ -189,8 +187,10 @@ protected void doDestroy() throws Exception {
   @Override
   protected void doDelete() throws Exception {
     // TODO: Make this a soft delete and cleanup later so it doesn't block for large repos.
-    try (StorageTx tx = openStorageTx(databaseInstanceProvider.get().acquire(), false)) {
+    try (StorageTx tx = openStorageTx(databaseInstanceProvider.get().acquire())) {
```

Comment:
`doDelete` is called when the storage facet is being deleted, after it has stopped. So if we want to call `txSupplier` here we'd either need to mark `txSupplier` as `@Guarded(by = {STARTED, STOPPED})` or remove the guard altogether - which would then leave it open to be called by other external code after the facet has stopped, which is not ideal.